db-instances.tf

```diff
@@ -1,12 +1,13 @@
 resource "aws_instance" "reut_db" {
-  count = 2
+  count         = 2
   ami           = data.aws_ami.ubuntu.id
   instance_type = "t2.medium"
-  user_data = file("nginx.sh")
-  key_name = "Reut"
-  subnet_id = aws_subnet.private_subnet.*.id[count.index]
-   tags = {
-      Name = "reut_db ${count.index}" 
-      purpose = "terraform hw exe4"  
-    }
-  }
\ No newline at end of file
+  key_name      = var.key_name
+  subnet_id     = aws_subnet.private_subnet.*.id[count.index]
+  vpc_security_group_ids = [aws_security_group.DB_instnaces_access.id]
```

Comment:
I created sg for DB instances
And I associate the DB instances with this new sg